lib/htmlWriter.js

```diff
@@ -3,18 +3,21 @@
 const {chunkToMarkup} = require("./markup");
 const {Transform} = require("stream");
 
-module.exports = function htmlWriter() {
-  return new Transform({
-    writableObjectMode: true,
-    transform(chunks, encoding, next) {
-      if (!chunks) return next();
-      chunks = Array.isArray(chunks) ? chunks : [chunks];
-      let markup = "";
-      for (const chunk of chunks) {
-        markup += chunkToMarkup(chunk);
-      }
-      return next(null, markup);
+class HtmlWriter extends Transform {
+  constructor() {
+    super({writableObjectMode: true});
+  }
+  _transform(chunks, encoding, next) {
```

Comment:
why `_transform` and not `transform`?

Reply:
part of the [api](https://nodejs.org/dist/latest-v14.x/docs/api/stream.html#stream_implementing_a_transform_stream)